src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/DraftApprenticeshipControllerTests/AddDraftApprenticeshipTests.cs

```diff
@@ -158,7 +141,7 @@ public AddDraftApprenticeshipTestsFixture()
 
             ModelMapper.Setup(m => m.Map<AddDraftApprenticeshipViewModel>(It.IsAny<AddDraftApprenticeshipRequest>())).ReturnsAsync(ViewModel);
 
-            LinkGenerator.Setup(g => g.CommitmentsLink(CohortDetailsUrl)).Returns(CohortDetailsUrl);
+            LinkGenerator.Setup(g => g.CommitmentsV2Link(CohortDetailsUrl)).Returns(CohortDetailsUrl);
```

Comment:
I think we should use Url.Action instead of CommitmentV2Link (If we use Url.Action, this setup may not be required.). But this is a suggestion. I will leave it upto you to decide/check if this is possible and it is not just too much extra work.